6.3 Fast Probabilistic Consensus.md

```diff
@@ -10,12 +10,23 @@ This procedure shall terminate locally if a node has not changed its opinion ove
 
 Unlike other voting-based consensus protocols, FPC uses a sequence of global random thresholds. This randomness makes FPC robust even in Byzantine environments. We refer to [FPC-BI: Fast Probabilistic Consensus within Byzantine Infrastructures](https://arxiv.org/abs/1905.10895), [Robustness and efficiency of leaderless probabilistic consensus protocols within Byzantine infrastructures](https://arxiv.org/abs/1911.08787), and [Fast Probabilistic Consensus with Weighted Votes](https://link.springer.com/chapter/10.1007/978-3-030-63089-8_24) for more details on FPC.
 
+
+The Fast Probabilistic Consensus Protocol specification depends on the following specifications:
+
++ [2.3 - Standard Payloads Layout](./2.2%20Standard%20Payloads%20Layout.md) 
++ [3.1 - Communication Models](./3.1%20Communication%20Models.md)
```

Comment:
We don't have this file yet, are we supposed to provide the link? (honest question, I don't know...)

Reply:
Yeah, no problem. It saves us future work.